### ISSUE TITLE
'unicode' object has no attribute 'items'

### ISSUE THREAD

**Issue by dhirschfeld** [mher/flower#301]
``` python
flower   -> 0.8.0-dev
software -> celery:3.1.15 (Cipater) kombu:3.0.23 py:2.7.8
            billiard:3.3.0.18 py-amqp:1.4.6
platform -> system:Windows arch:64bit, WindowsPE imp:CPython
loader   -> celery.loaders.default.Loader
settings -> transport:amqp results:disabled



Traceback (most recent call last):
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\web.py", line 1334, in _execute
    result = yield result
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\gen.py", line 628, in run
    value = future.result()
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\concurrent.py", line 109, in result
    raise_exc_info(self._exc_info)
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\gen.py", line 633, in run
    yielded = self.gen.send(value)
  File "C:\Python\envs\continuous_integration\lib\site-packages\flower\views\workers.py", line 33, in get
    self.render("worker.html", worker=worker)
  File "C:\Python\envs\continuous_integration\lib\site-packages\flower\views\__init__.py", line 21, in render
    super(BaseHandler, self).render(*args, **kwargs)
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\web.py", line 665, in render
    html = self.render_string(template_name, **kwargs)
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\web.py", line 772, in render_string
    return t.generate(**namespace)
  File "C:\Python\envs\continuous_integration\lib\site-packages\tornado\template.py", line 278, in generate
    return execute()
  File "worker_html.generated.py", line 397, in _tt_execute
    for name, value in worker['stats']['rusage'].items():  # worker.html:337 (via base.html:48)
AttributeError: 'unicode' object has no attribute 'items'
```

**Comment by Thhhza** [comment]
I am getting this error again because the change made by mher above has been reverted?

https://github.com/mher/flower/commit/6ae80baf4832d2c5090c067815a7bd68acc80a98

**Comment by mher** [comment]
Thanks, fixed by #372
